search.py

```diff
@@ -324,27 +324,34 @@ def text_box_search_folder(path, tag, search_term):
         string_list = []
         for element in tb_search_output_array:
             string_list.append(' '.join(str(e) for e in get_title(file)) + " by " +
-                               ' '.join(str(e) for e in get_creator(file)) + ": " + element)
+                               ' '.join(str(e) for e in get_creator(file)) + element)
             text_box_array.append(string_list)
     return text_box_array
 
 
 def snippet_search_folder(path, input_tree):
```

Comment:
This might not be the right PR for this, but are these function returning strings? Why not return something more structured, ie. a list of a [`namedtuple`](https://docs.python.org/dev/library/collections.html#collections.namedtuple)?